app/index.js

```diff
@@ -46,10 +46,17 @@ module.exports = JhipsterGenerator.extend({
             type: Boolean,
             defaults: true
         });
+        // This method adds support for a `--skip-entities` flag
+        this.option('skip-entities', {
```

Comment:
Well, it seems natural to me to update the entities after a jhipster upgrade since you will most probably have compatibility issues if you don't.
I'm not against adding it as a question though.
@jdubois what is your opinion ?